lib/rbs/prototype/rb.rb

```diff
@@ -408,9 +420,11 @@ def literal_to_type(node)
 
       def types_to_union_type(types)
         return untyped if types.empty?
-        return untyped if types.include?(untyped)
```

Comment:
Previously `Something | untyped` is squashed to `untyped`. But I think `Something | untyped` is useful for humans when reviewing the generated types. (The original idea is the type profiler.)
So it will not be squashed by this pull request.